api/server.js

```diff
@@ -1,3 +1,5 @@
+require('dotenv').config();
```

Comment:
In bigger applications, and depending on your exact situation, this'll often get called even sooner. For applications using knex, that's often in the knexfile.